bert/model.py

```diff
@@ -214,19 +215,29 @@ def run(bert_model: str, mlx_model: str):
         "A second string",
         "This is another string.",
     ]
-    
+
     tokens = tokenizer(batch, return_tensors="np", padding=True)
     tokens = {key: mx.array(v) for key, v in tokens.items()}
 
-    mlx_output, mlx_pooled = model(**tokens)
-    mlx_output = numpy.array(mlx_output)
-    mlx_pooled = numpy.array(mlx_pooled)
-
-    print("MLX BERT:")
-    print(mlx_output)
-
-    print("\n\nMLX Pooled:")
-    print(mlx_pooled[0, :20])
+    vs = model_configs[bert_model].vocab_size
```

Comment:
Were the BERT changes supposed to be in this PR.

Reply:
Ugh good catch, that was not intentional.. just doing some timing for myself :).